accept start transaction statements (#11323)
crate/crate: docs/interfaces/postgres.rst

    +-----------------------+
    SHOW 1 row in set (... sec)

BEGIN/START/COMMIT statements
-----------------------------

For compatibility with clients that use the Postgres wire protocol, such as the
Golang lib/pq and pgx drivers, the full PostgreSQL syntax of the
:ref:`BEGIN <ref-begin>`, :ref:`START <ref-start>` and :ref:`COMMIT <ref-commit>`

> statements is implemented, for example::

Reviewer: yep

> statements is implemented, for example::

Author: added

> statements are implemented, for example::

Reviewer 2: how about:

    For compatibility with clients that use the PostgresSQL wire protocol (e.g.,
    the Golang lib/pq and pgx drivers), CrateDB will accept the :ref:`BEGIN
    <ref-begin>`, :ref:`COMMIT <ref-commit>`, and :ref:`START TRASNACTION
    <sql-start-transaction>` statements. For example::

changes here:

- fixed "Postgres" (as above)
- tidied up the grammar/phrasing (I realize this is the text you are modifying, but while we're in here we may as well tidy it up a little) :)
- expanded ``START`` to ``START TRANSACTION`` so we're using the statement in full (better to be as explicit as possible)
- reordered the list of statements to match the list of statements that appear later down this sectiobn
- I reworded things slightly so that we say that CrateDB "accepts" the statements. I think "implements" and "supports" may confuse matters slightly because both of them _may_ give the impression that these statements are something more than a no-op

    cr> COMMIT
    COMMIT OK, 0 rows affected  (... sec)

Since CrateDB does not support transactions, the ``COMMIT`` and ``BEGIN/START``

> statement and any of its parameters are ignored.

Author: statement > statements right?

Reviewer: yes :+1:

Author: added thanks!

    cr> COMMIT
    COMMIT OK, 0 rows affected  (... sec)

> Since CrateDB does not support transactions, the ``COMMIT`` and ``BEGIN/START``
> statements and any of its parameters are ignored.

Reviewer 2: ```suggestion
CrateDB will silently ignore the ``COMMIT TRANSACTION``, ``BEGIN TRANSACTION``,
and ``START TRANSACTION statements and all respective parameters.
```

reworded this to use the [active voice](https://www.grammarly.com/blog/active-vs-passive-voice/). I also expanded the statement literals for precision and clarity

I removed the "since" part because I think its redundant given the rest of this subsection
